package.json

```diff
@@ -15,12 +15,39 @@
   },
   "author": "",
   "license": "ISC",
+  "jest": {
+    "moduleFileExtensions": [
+      "es6",
+      "js"
+    ],
+    "verbose": true,
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
+      "\\.(scss|css|less)$": "identity-obj-proxy"
+    },
+    "collectCoverage": true,
+    "unmockedModulePathPatterns": [
+      "react"
+    ],
+    "setupTestFrameworkScriptFile": "mock-local-storage"
+  },
   "bugs": {
     "url": "https://github.com/Lv-237-NodeJS/da-mi/issues"
   },
   "homepage": "https://github.com/Lv-237-NodeJS/da-mi#readme",
   "devDependencies": {
+    "babel-jest": "^20.0.3",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
i took it from jest guide for adjusting work. If you know another way, could you tell me?